### PR TITLE
chore(renovate): fix GOLANG_VERSION and GOLANGCI_LINT

### PR DESCRIPTION
**Pull request by phisco**

### CODE REVIEW

**Review by mnencia — APPROVED** [review]

File: internal/cmd/manager/controller/controller.go [hunk 1]
```diff
@@ -41,7 +42,6 @@ import (
 	"github.com/cloudnative-pg/cloudnative-pg/pkg/multicache"
 	"github.com/cloudnative-pg/cloudnative-pg/pkg/utils"
 	"github.com/cloudnative-pg/cloudnative-pg/pkg/versions"
-	// +kubebuilder:scaffold:imports
```
Comment (sxd) [inline internal/cmd/manager/controller/controller.go:44]: This requires the new version of golangci right? So everyone should be running the new version ?

**Review by sxd — APPROVED** [review]

File: internal/cmd/manager/controller/controller.go (hunk 1 above)
Comment (mnencia) [reply internal/cmd/manager/controller/controller.go:44]: No, it doesn't. It's also suitable for the old version.

Comment (sxd) [reply internal/cmd/manager/controller/controller.go:44]: Nice!! Then let's merge it :)

**Comment by mnencia** [comment]
@phisco has more changes to do (we do not want the full versiond but only Major.Minor)

**Pull request merged**